### PR TITLE
Configure embedded database with properties

### PR DESCRIPTION
**Pull request by akalotkin**
Spring embedded database support isn't very configurable at the moment.
These changes related to SPR-12455.

I have signed and agree to the terms of the SpringSource Individual
Contributor License Agreement.

### CODE REVIEW

File: spring-jdbc/src/main/resources/org/springframework/jdbc/config/spring-jdbc-4.2.xsd [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		elementFormDefault="qualified"
 		attributeFormDefault="unqualified">
 
-	<xsd:import namespace="http://www.springframework.org/schema/beans" schemaLocation="http://www.springframework.org/schema/beans/spring-beans-4.2.xsd"/>
+	<xsd:import namespace="http://www.springframework.org/schema/beans" schemaLocation="file:////Users/akalotkin/dev/projects/opensource/spring/spring-beans/src/main/resources/org/springframework/beans/factory/xml/spring-beans-4.2.xsd"/>
```
Comment (sbrannen) [inline spring-jdbc/src/main/resources/org/springframework/jdbc/config/spring-jdbc-4.2.xsd]: Please do not reference resources in your local file system.

File: spring-jdbc/src/test/java/org/springframework/jdbc/config/JdbcNamespaceIntegrationTests.java [hunk 2]
```diff
@@ -23,9 +24,10 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
-
```
Comment (sbrannen) [inline spring-jdbc/src/test/java/org/springframework/jdbc/config/JdbcNamespaceIntegrationTests.java]: Please retain blank link between org.junit and org.springframework imports.

**Comment by sbrannen** [comment]
For any new classes or new methods in existing classes, please add `@since 4.3` tags.

**Comment by akalotkin** [comment]
I've fixed all issues.

Thanks.

**Comment by sbrannen** [comment]
Great!

Since Spring 4.2 GA is now just around the corner, I won't have time to review your pull request fully until work on 4.3 begins.

But... your work will not be forgotten. :wink:

**Comment by pivotal-issuemaster** [comment]
@akalotkin Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=833)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=833) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by sbrannen** [comment]
Due to the age of this PR and lack of recent interest in #17060, I am closing this PR.

Please note, however, that this issue will be taken into further consideration if there is further community interested expressed in #17060.

@akalotkin, thanks for submitting the PR and apologies for not getting back to you sooner.

**Pull request closed without merging**